metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/README.md

```diff
@@ -0,0 +1,3 @@
+# bro Parser
+
+This is the bro parser
```

Comment:
What do each of the 3 projects for each parser do?  Trying to understand why each parser extension can't just live in a single project.

For example, why 3 projects for the Bro parser?
```
/metron-parser-bro-extension
-----> /metron-parser-bro
-----> /metron-parser-bro-assembly
-----> /metron-parser-bro-bundle
```